src/locales/es/translation.json

```diff
@@ -1,5 +1,10 @@
 {
   "title": "[es] Project Title",
+  "department": {
```

Comment:
You name it `department` instead of `agency` here.

Reply:
good catch